worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Controller.scala

```diff
@@ -324,6 +324,42 @@ private[deploy] class Controller(
       return
     }
 
+    val shuffleCommitTimeout = conf.workerShuffleCommitTimeout
+
+    shuffleCommitInfos.putIfAbsent(shuffleKey, new CommitInfo(null, CommitInfo.COMMIT_NOTSTARTED))
+    val status = shuffleCommitInfos.get(shuffleKey)
```

Comment:
rename status to shuffleCommitInfo would be better

Reply:
done